ansible/roles/polkadot-common/tasks/main.yml

```diff
@@ -64,6 +64,7 @@
   systemd:
     name: polkadot.service
     state: started
+    daemon_reload: yes
```

Comment:
We don't need daemon_reload for the dummy service, this is only started the first time to generate the network peer ID and in this case there's no previous configuration present to reload.
```suggestion
```

Reply:
@fgimenez I added it because if there's something that goes wrong in a previous playbook run, this would fail here. I was messing around with the unit file, so this might be a very specific issue. Up to you.

Reply:
Yeah, in any case I think it would be better to discuss this in a separate PR, maybe after trying to identify when this concrete issue can happen (haven't seen it so far). It doesn't seem related to the rest of the changes in this PR.